wl-clipboard-tools/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "wl-clipboard-tools"
+version = "0.4.1" # remember to update html_root_url
+authors = ["Ivan Molodetskikh <yalterz@gmail.com>"]
+description = "Access to the Wayland clipboard for terminal and other window-less applications."
+edition = "2018"
+license = "MIT/Apache-2.0"
+
+readme = "README.md"
+documentation = "https://docs.rs/wl-clipboard-rs"
+repository = "https://github.com/YaLTeR/wl-clipboard-rs"
+keywords = ["wayland", "clipboard"]
+categories = ["command-line-utilities"]
+
+[badges]
+travis-ci = { repository = "YaLTeR/wl-clipboard-rs" }
+
+# [profile.dev]
+# panic = "abort"
+
+[dependencies]
+wl-clipboard-rs = { path = "../" }
+derive-new = "0.5"
+derive_more = "0.99"
+exitfailure = "0.5"
+failure = "0.1"
+libc = "0.2"
+log = "0.4"
+mime_guess = "2"
+nix = "0.18"
+stderrlog = "0.4"
+structopt = { version = "0.3", features = ["wrap_help"]}
+os_pipe = "0.9"
+tempfile = "3"
+tree_magic_mini = "3.0.2"
+wayland-client = "0.27"
+
+[dependencies.wayland-protocols]
+version = "0.27"
+features = ["client", "unstable_protocols"]
+
+[dev-dependencies]
+wayland-server = "0.27"
+
+[dev-dependencies.wayland-protocols]
+version = "0.27"
+features = ["server", "unstable_protocols"]
+
+[features]
+# Link to libwayland-client.so instead of using the Rust implementation.
+native_lib = [
+    "wayland-client/use_system_lib",
```

Comment:
Instead of wayland-client, this should depend on `wl-clipboard-rs/native_lib` I believe.